.github/workflows/test.yml

```
@@ -0,0 +1,14 @@
+name: test
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm ci
```

```suggestion
      - run: npm install
```

Had troubles with `npm ci` in the past related to native modules.

---

Let's see if this is still a thing with latest npm.

---

You'll probably not run into it but I still like to have this precaution. I don't trust `npm ci` 😉